client/boot/index.js

```diff
@@ -208,6 +208,10 @@ function reduxStoreReady( reduxStore ) {
 		require( 'lib/network-connection' ).init( reduxStore );
 	}
 
+	if ( config.isEnabled( 'css-hot-reload' ) ) {
```

Comment:
@blowery @mtias: It's not directly related to this PR, but in general to the approach we take to the development tools we use. 

We should check if we can refactor our code to put the logic that enables any kind of dev tools inside `loadDevModulesAndBoot` function and make sure it's loaded as a part of `devmodules` webpack section. In addition we should make sure we don't generate this section for production build in a similar fashion as I proposed in #5516.